web/src/helpers.tsx

```diff
@@ -45,17 +46,17 @@ export const getMenuData = (): MenuDataItem[] => {
     {
       name: 'service',
       path: '/service/list',
-      icon: <IconFont name="iconconsumer" />,
+      icon: <IconFont name="iconservice" />,
     },
     {
       name: 'consumer',
       path: '/consumer/list',
-      icon: <IconFont name="iconconsumer" />,
+      icon: <IconFont name="iconconsumer1" />,
```

Comment:
it seems that `iconconsumer` is used, how about remove the old `iconconsumer`, and rename the new one `iconconsumer1` to `iconconsumer`, that would be more clear

Reply:
Sure.